tests/textcase/lsp/lsp_spec.lua

```diff
@@ -1 +1,40 @@
-describe("LSP", function() end)
+local test_helpers = require("tests.test_helpers")
+local textcase = require("textcase")
+
+describe("LSP", function()
+  describe("Rename", function()
+    before_each(function()
+      textcase.setup({})
+    end)
+
+    it("Should be triggered on keybinding", function()
+      local path = "./tests/textcase/lsp/fixtures/component-camel-case.tsx"
+      local cmd = " silent exe 'e " .. path .. "'"
+      vim.cmd(cmd)
+      vim.bo.filetype = "typescriptreact"
+      -- allow tsserver start
+
+      local ts_server_started = false
+      test_helpers.wait_for(30 * 1000, function()
+        vim.cmd("LspInfo")
+        ts_server_started = not not vim.fn.search("1 client(s) attached to this buffer")
+        test_helpers.execute_keys("q")
+        return ts_server_started
+      end)
+
+      assert.is.truthy(ts_server_started)
+      vim.wait(1000, function() end)
+
+      test_helpers.execute_keys("/variableToBeTested<CR>gaS")
+      local content = nil
+      M.wait_for(5 * 1000, function()
```

Comment:
So this is an interesting one: M.wait_for is not part of this file anymore. However, it still works! And it's calling the method from `test_helpers`. 🤔 Lua has some strange implementations ... I would still change it.

Reply:
Ah it was me when I created the `test_helpers`. It's missing a `local` before `M = {}`. Can you add this please so this here fails?

Reply:
Not so strange after all ^^